infra-as-code/bicep/machinelearning.bicep

```diff
@@ -189,8 +183,6 @@ resource aiHub 'Microsoft.MachineLearningServices/workspaces@2024-07-01-preview'
       authType: 'AAD'
       category: 'AzureOpenAI'
       isSharedToAll: true
-      useWorkspaceManagedIdentity: true
-      peRequirement: 'NotRequired'

```

Comment:
Why did we remove `peRequirement` -- this was us being explicit for the "basic" implementation here to show that we don't expect a private endpoint based connection.

Reply:
still present